src/adapters/refract/transformResource.coffee

```diff
@@ -147,19 +147,20 @@ module.exports = (resourceElement, location, options) ->
 
         requests.push(request)
 
-      response = new blueprintApi.Response({
-        status: _.chain(httpResponse).get('attributes.statusCode').contentOrValue().value()
-        description: httpResponseDescription.raw
-        htmlDescription: httpResponseDescription.html
-        headers: getHeaders(httpResponse)
-        # reference
-        body: trimLastNewline(if _.content(httpResponseBody) then _.content(httpResponseBody) else '')
-        schema: trimLastNewline(if _.content(httpResponseBodySchemas) then _.content(httpResponseBodySchemas) else '')
-        # exampleId
-        attributes: responseAttributes
-      })
+      if httpResponse?.content.length or (not _.isEmpty(httpResponse?.attributes)) or (not _.isEmpty(httpResponse?.meta))
```

Comment:
Why would we care if the meta is not empty? We don't use any values from meta to create the response.

A response having a meta means we will create a response here, but the meta value doesn't make it any different to the old behaviour of having an empty response.